pkgs/by-name/rs/rsshub/0001-fix-git-hash.patch

```diff
@@ -0,0 +1,30 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
Since the patch affects only a single file, wouldn't it be better as a simple diff?